### PR TITLE
CHANGELOG: Add meta-nilrt #626, #628, #640

### PR DESCRIPTION
**Pull request by chaitu236**
Update changelog with meta-nilrt changes in last couple of months.

### Testing
- [x] Updated lines display correctly in GitHub UI and links work.

### CODE REVIEW

**Review by amstewart — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ The evergreen, canonical changelog for *all NILRT branches* can be [found here](
 To see changes to each individual package in the core feed, check out the [feed changelog](/docs/feed-changelog.md)
 
 ## Latest Updates
-* nilrt: 245
-* meta-nilrt: 623
+* nilrt: 254
```
Comment (chaitu236) [inline CHANGELOG.md:14]: @amstewart: I can understand the point of tracking last PR of `meta-nilrt` here but why `nilrt`? We don't usually make any distro relevant changes in `nilrt` repo.

Comment (amstewart) [reply CHANGELOG.md:14]: We do sometimes, especially when we are doing a distro rebase.

Comment (chaitu236) [reply CHANGELOG.md:14]: You mean adding/removing layers and that kind of stuff?

Comment (amstewart) [reply CHANGELOG.md:14]: Sure. Or changes to tooling or host requirements.

**Pull request merged**